verify/verify-pylint.sh

```diff
@@ -20,6 +20,7 @@ set -o pipefail
 export PYLINTHOME=$(mktemp -d)
 pylint="$(dirname $0)/pylint_bin"
 
+pip install --upgrade requests pyyaml
```

Comment:
Otherwise lint complains when it tries to import this module. I want to eventually delete the requirements.txt file but cannot in this PR due to reasons I dont yet understand.